src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -80,35 +80,35 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.Azure.WebJobs.Extensions, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Microsoft.Azure.WebJobs.Extensions.2.0.0-beta2-10403\lib\net45\Microsoft.Azure.WebJobs.Extensions.dll</HintPath>
+      <HintPath>C:\code\Git\Azure\azure-webjobs-sdk-extensions\test\WebJobs.Extensions.Tests\bin\Debug\Microsoft.Azure.WebJobs.Extensions.dll</HintPath>
```

Comment:
Ignore these proj file changes - I won't check these in. These are only for my own local e2e testing